src/main/target/WORMFC/target.h

```diff
@@ -0,0 +1,140 @@
+
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+
+#define BEEPER                  	PB14
+#define BEEPER_INVERTED
+
+#define INVERTER_PIN_UART6      	PB13 // PC0 used as inverter select GPIO
+#define INVERTER_PIN_UART3      	PB12 // PC0 used as inverter select GPIO
+
+// MPU6500 interrupt
+#define USE_EXTI
+#define MPU_INT_EXTI            	PC4
+#define USE_MPU_DATA_READY_SIGNAL
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
+
+#define ICM20689_CS_PIN          PA4
+#define ICM20689_SPI_INSTANCE    SPI1
+
+#define MPU6500_CS_PIN          	PA4
+#define MPU6500_SPI_INSTANCE    	SPI1
+
+// ACC section -- start
+#define ACC
+
+#define USE_ACC_SPI_ICM20689
+#define ACC_ICM20689_ALIGN       CW180_DEG_FLIP
+
+#define USE_ACC_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW180_DEG_FLIP
+// ACC section -- end
+
+// GYRO section -- start
+#define GYRO
+
+#define USE_GYRO_SPI_ICM20689
+#define GYRO_ICM20689_ALIGN      CW180_DEG_FLIP
+
+#define USE_GYRO_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define GYRO_MPU6500_ALIGN      CW180_DEG_FLIP
+// GYRO section -- end
+
+#define USE_VCP
+#define VBUS_SENSING_PIN        PA9
+#define VBUS_SENSING_ENABLED
+
+#define USE_UART1
+#define UART1_RX_PIN            PB7
+#define UART1_TX_PIN            PB6
+
+#define USE_UART3
+#define UART3_RX_PIN            PB11
+#define UART3_TX_PIN            PB10
+
+#define USE_UART6
+#define UART6_RX_PIN            PC7
+#define UART6_TX_PIN            PC6
+
+#define SERIAL_PORT_COUNT       4 //VCP, USART1, USART3, USART6, SOFTSERIAL x 2
+
+#define USE_ESCSERIAL
+#define ESCSERIAL_TIMER_TX_HARDWARE 0 // PWM 1
+
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+
+#define USE_SPI_DEVICE_3
+#define SPI3_SCK_PIN            PB3
+#define SPI3_MISO_PIN           PB4
+#define SPI3_MOSI_PIN           PB5
+
+#define USE_OSD
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE    SPI3
+#define MAX7456_SPI_CS_PIN      PC0
+#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD) // 10MHz
+#define MAX7456_RESTORE_CLK     (SPI_CLOCK_FAST)
+
+#define BARO
+#define USE_BARO
+#define USE_BARO_LPS
+#define LPS_SPI_INSTANCE SPI3
+#define LPS_CS_PIN PB8
+
+// #define MAX7456_DMA_CHANNEL_TX              DMA1_Stream5
+// #define MAX7456_DMA_CHANNEL_RX              DMA1_Stream0
+// #define MAX7456_DMA_IRQ_HANDLER_ID          DMA1_ST0_HANDLER
+
+#define USE_ADC
+#define BOARD_HAS_VOLTAGE_DIVIDER
+#define VBAT_ADC_PIN            PC1
+#define RSSI_ADC_PIN            PC2
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+#define SERIALRX_PROVIDER       SERIALRX_SBUS
+
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
+
+#define USE_SDCARD
+#define USE_SDCARD_SDIO
+#define SDIO_DMA
+//This is not used on SDIO
+#define SDCARD_SPI_CS_PIN PB15
+#define SDCARD_DETECT_PIN PB15
+
+#define TARGET_IO_PORTA         0xffff
+#define TARGET_IO_PORTB         0xffff
+#define TARGET_IO_PORTC         0xffff
+#define TARGET_IO_PORTD         0xffff
```

Comment:
Only a few pins are available at Port D for the STM32F405. Also see other F405 targets.

Reply:
Yeah correct should be 0X11, I had some issues with CMD pin and thought it was related to this. Turned out it wasn’t... I forgot to clean it later